src/ResourceManager/Compute/AzureRM.Compute.ManagedService.psd1

```diff
@@ -0,0 +1,126 @@
+#
```

Comment:
This should be in its own directory (Compute.ManagedService)

Reply:
They were in different folders from the initial commits...

Reply:
moved.